pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -440,21 +432,6 @@ func TestReconcile(t *testing.T) {
 			Eventf(corev1.EventTypeWarning, "InternalError",
 				`error scaling target: failed to apply scale 11 to scale target test-revision-deployment: inducing failure for patch deployments`),
 		},
-	}, {
-		Name: "can't read endpoints",
```

Comment:
Is it necessary to have a test covering similar case for that pods can't be read?